test/e2e/cassandra_test.go

```diff
@@ -22,9 +22,10 @@ func TestCassandra(t *testing.T) {
 
 	t.Run("scaling up", CassandraScalingUp)
 	t.Run("scaling down", CassandraScalingDown)
+	t.Run("minimal spec test", MinimalCassandraSpec)
 }
 
-func deployCassandra(t *testing.T, nodes, racks int32) (*framework.TestCtx, *framework.Framework, *v1alpha1.CassandraDataCenter, string) {
+func deployCassandra(t *testing.T, nodes int32, racks []v1alpha1.Rack) (*framework.TestCtx, *framework.Framework, *v1alpha1.CassandraDataCenter, string) {
```

Comment:
weird import naming, let's rename v1alpha1 to something more meaningful like `cassandraoperatorv1alpha1` ;-)